pubspec.yaml

```diff
@@ -14,7 +14,10 @@ dependencies:
   app_installer: ^1.1.0
   collection: ^1.17.0
   cross_connectivity: ^3.0.5
-  cr_file_saver: ^0.0.2+1
+  cr_file_saver:
+    git:
+      url: https://github.com/dhruvanbhalara/cr_file_saver.git
```

Comment:
What is the purpose of that commit ref? why can't latest upstream be used?

Reply:
The commit points to a branch that includes changes, assuming fixes are not accepted by upstream; for that reason, I think it is better to try to PR it to upstream instead of using the fork.

Reply:
I checked the main plugin I didn't see any updates for a long time so I added a reference of the forked repo I'll create PR against the main repo of the package and check if it gets merged or not.

Reply:
> What is the purpose of that commit ref? why can't latest upstream be used?

I've raised the PR in the main repo required to fix the mentioned issue in the PR description, if you want to merge this PR feel free to do it.